pyrobosim_ros_gym/train.py

```diff
@@ -89,23 +73,22 @@
     if args.model_type == "DQN":
         policy_kwargs = {
             "activation_fn": nn.ReLU,
-            "net_arch": [64, 64],
+            "net_arch": [8, 4],
```

Comment:
these hard-coded defaults are going to hurt the banana env performance, please make sure they're specific to the selected environment